public/projects/index.html

```diff
@@ -76,27 +76,31 @@
                   <a class="nav__dropdown__papers__item">Nitro V2 (Coming Soon!)</a>
                 </li>
                 <li>
-                  <a class="nav__dropdown__papers__item" target="_blank" href="https://magmo.com/force-move-games.pdf">Force
+                  <a class="nav__dropdown__papers__item" target="_blank"
```

Comment:
I think many of these changes to this file are formatting changes (to the html) which won't have any affect on the way the page looks, but do make code review a little more painful. Let's set up `prettier` in this repo so that we force all code to be formatted in the same way? That should cut down on the noise in PRs.